.github/workflows/release.yml

```diff
@@ -127,22 +127,28 @@ jobs:
 
       - name: Get PR Description
```

Comment:
- Consider handling the case where `context.payload.pull_request` is not defined more explicitly to avoid potential errors.